packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -10,5 +10,6 @@
   * Added blank cell support.
   * Added ability to allow consumers to specify the sort indicator and error state for columns.
 
-## 0.1
-  * Initial stable release.
```

Comment:
Why is this being removed? Is it due to a merge conflict? This needs to be added back and any new entries should be above.

Reply:
Merging error. Fixed